tensorflow/python/ops/nn.py

```diff
@@ -1029,7 +1029,7 @@ def fused_batch_norm(x, scale, offset,  # pylint: disable=invalid-name
       offset,
       mean,
       variance,
-      epsilon=epsilon,
+      epsilon=epsilon + 1e-12,
```

Comment:
Not sure if this will introduce subtle failures in testing due to the introduced inconsistency in epsilon. How about adding this inside the function? epsilon = epsilon if epsilon > 1e-5 else epsilon + 1e-12